regexp-injection/LauraBeatris_pokedex-nextjs_search.ts

```
@@ -0,0 +1,18 @@
+export default (req: NextApiRequest, res: NextApiResponse): void => {
+  const filterQueryName = req.query.name
+    ? new RegExp(req.query.name as string, "i")
+    : /.*/;
+
+  const filteredPokemons = filterPokemons(pokemonData, filterQueryName);
+
+  res.setHeader("Content-Type", "application/json");
+
+  return res
+    .status(200)
+    .send(JSON.stringify(filteredPokemons));
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)